cmse/cmse.md

```diff
@@ -1808,12 +1810,14 @@ foo:
 __acle_se_foo:
     @ 1: if called from non-secure reserve secure stack space for the arguments
     tst     lr, #1
-    it      eq
+    ittt    eq
     subeq   sp, sp, #8
+    @ 3: save FPCXTNS if called from non-secure. LR is stacked only for alignment purposes
```

Comment:
couple of points about this:
1) You shouldn't make this conditional. Just detecting whether we've been called from NS isn't enough to get the correct behaviour as there's some nasty corner cases. To make it easier instructions that access FPCXT_NS already have all the special behaviour built into them, but for that to work they must be executed unconditionally
2) There's no need to use a VMRS followed by a PUSH. Since we expected this to be a common code pattern we created a VSTR (system register) instruction that can access the FPCXT_* payloads. The addressing modes provides by this instruction are flexible enough that you can also preserve the 64bit SP alignment